TILELANG_CLEAR_CACHE is an abandoned environment variable

https://github.com/tile-ai/tilelang/blob/a4ab99e4664de377bb6bc035cc8ab13a8d735733/tilelang/cache/__init__.py#L110

The only use of TILELLANG_CLEAR-CACHE in the entire code repository is the clear_cache() method that triggers an error, and there are no documents, tests, scripts, or CI configurations referencing it.

The error print like:
```txt
  File "/mnt/workspace/developer/workspace/git/github/erhsh/tilelang.lower-trace.v1/tilelang/cache/__init__.py", line 103, in clear_cache
    raise RuntimeError(
RuntimeError: tilelang.clear_cache() is disabled because deleting the cache directory is dangerous. If you accept the risk, remove it manually with `rm -rf '/home/developer/.tilelang/cache'`.
[ERROR] 2026-06-18-16:16:52 (PID:1135809, Device:-1, RankID:-1) ERR99999 UNKNOWN applicaiton exception
```

--- a/tilelang/__init__.py
+++ b/tilelang/__init__.py
@@ -185,7 +185,6 @@ if not env.is_light_import():
 
     from .jit import jit, JITKernel, compile, par_compile  # noqa: F401
     from .profiler import Profiler  # noqa: F401
-    from .cache import clear_cache  # noqa: F401
     from .utils import (
         TensorSupplyType,  # noqa: F401
         deprecated,  # noqa: F401

--- a/tilelang/cache/__init__.py
+++ b/tilelang/cache/__init__.py
@@ -90,22 +90,3 @@ def cached(
         pass_configs=pass_configs,
         compile_flags=compile_flags,
     )
-
-
-def clear_cache():
-    """
-    Disabled helper that previously removed the entire kernel cache.
-
-    Raises:
-        RuntimeError: Always raised to warn users to clear the cache manually.
-    """
-    cache_dir = env.TILELANG_CACHE_DIR
-    raise RuntimeError(
-        "tilelang.clear_cache() is disabled because deleting the cache directory "
-        "is dangerous. If you accept the risk, remove it manually with "
-        f"`rm -rf '{cache_dir}'`."
-    )
-
-
-if env.TILELANG_CLEAR_CACHE.lower() in ("1", "true", "yes", "on"):
-    clear_cache()

--- a/tilelang/env.py
+++ b/tilelang/env.py
@@ -322,7 +322,6 @@ class Environment:
     TILELANG_DISABLE_CACHE = EnvVar(
         "TILELANG_DISABLE_CACHE", "0"
     )  # disable kernel cache, usually for unit testing / debugging, high priority
-    TILELANG_CLEAR_CACHE = EnvVar("TILELANG_CLEAR_CACHE", "0")  # DEPRECATED! clear cache automatically if set
     TILELANG_CLEANUP_TEMP_FILES = EnvVar(
         "TILELANG_CLEANUP_TEMP_FILES", "1"
     )  # cleanup temporary compiler files/dirs after compilation (set to 0 to keep for debugging)

